src/views/ApartmentApp/components/StudentApplication/index.js

```diff
@@ -378,302 +375,283 @@ export default class StudentApplication extends Component {
+    setChangeEditorDialogOpen(false);
+    setSubmitDialogOpen(false);
+  };
+
+  const handleCloseDialog = (event, reason) => {
     // Prevent the dialog box from closing if the user clicks outside the dialog box
     if (reason === 'clickaway') {
       return;
     }
-    this.handleCloseOkay();
+    handleCloseOkay();
   };
 
-  handleCloseOkay = () => {
-    this.setState({ submitDialogOpen: false, editDialogOpen: false });
+  const handleCloseSnackbar = (event, reason) => {
+    // Prevent the snackbar from closing if the user clicks outside the snackbar
+    if (reason === 'clickaway') {
+      return;
+    }
+    setSnackbarOpen(false);
   };
 
-  render() {
-    if (this.props.authentication) {
-      /**
-       * Used to re-render the page when the network connection changes.
-       * this.state.network is compared to the message received to prevent
-       * multiple re-renders that creates extreme performance lost.
-       * The origin of the message is checked to prevent cross-site scripting attacks
-       */
-      window.addEventListener('message', (event) => {
-        if (
-          event.data === 'online' &&
-          this.state.network === 'offline' &&
-          event.origin === window.location.origin
-        ) {
-          this.setState({ network: 'online' });
-        } else if (
-          event.data === 'offline' &&
-          this.state.network === 'online' &&
-          event.origin === window.location.origin
-        ) {
-          this.setState({ network: 'offline' });
-        }
-      });
+  const changeEditorAlertText = (
+    <span>
+      If you change the application editor, you will no longer be able to edit this application
+      yourself.
+      <br />
+      Are you sure you want to change the application editor?
+    </span>
+  );
 
-      const primaryApplicantAlertText = (
-        <span>
-          If you change the application editor, you will no longer be able to edit this application
-          yourself.
-          <br />
-          Are you sure you want to change the application editor?
-        </span>
-      );
+  const submitAlertText = (
+    <span>
+      This feature is not yet implemented.
+      <br />
+      Clicking the "Accept" button will hide the application cards.
+    </span>
+  );
 
-      /**
-       * Gets status of current network connection for online/offline rendering
-       * Defaults to online in case of PWA not being possible
-       */
-      const networkStatus = JSON.parse(localStorage.getItem('network-status')) || 'online';
-
-      if (networkStatus === 'online' && this.props.authentication) {
-        return (
-          <div>
-            {this.state.loading ? (
-              <GordonLoader />
-            ) : (
-              <div className="apartment-application">
-                <Grid container justify="center" spacing={2}>
+  return (
+    <div>
+      {loading ? (
+        <GordonLoader />
+      ) : (
+        <div className="apartment-application">
```

Comment:
Rather than wrapping in an empty div, could you just return the loader if loading, else return the application?
```suggestion
  if (loading) {
      return <GordonLoader />
  } else {
  return (
        <div className="apartment-application">
```

Reply:
I spotted this during my second review of the code, while making some simply bug fixes